src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -179,6 +179,31 @@ export function transformRelatedDisabilities(object, claimedConditions) {
     .map(key => key.toLowerCase());
 }
 
+/**
+ * Converts the treatment date range into an array of objects from just an object
+ *
+ * @param {object} treatmentDateRange object containing from/to date range
+ * @returns {array} of treatmentDateRange's
+ */
+const transformDateRange = treatmentDateRange => [treatmentDateRange];
+
+/**
+ * Cycles through the list of provider facilities and performs transformations on each property as needed
+ * @param {array} providerFacility array of objects being transformed
+ * @returns {object} containing the new Provider Facility structure
+ */
+const transformProviderFacilities = providerFacilities => {
+  const newProviderFacilities = providerFacilities.map(facility =>
```

Comment:
no need to store the mapped array in a const; you can just do `return providerFacilities.map(...`